server/models/userModel.js

```diff
@@ -27,11 +27,7 @@ const userSchema = mongoose.Schema(
     platforms: {
```

Comment:
I was thinking for the demo, maybe we could set them all to true by default! But we can discuss in standup